SwiftOverpassDemo/SwiftOverpassDemo/ViewController.swift

```diff
@@ -15,11 +15,12 @@ class ViewController: UIViewController {
         super.viewDidLoad()
         // Do any additional setup after loading the view, typically from a nib.
         
-        let query = SwiftOverpass.query(type: .node)
-        query.hasTag("name", equals: "Schloss Neuschwanstein")
+        let node = NodeQuery()
```

Comment:
Great! Not only is it shorter than the original approach, but it also prevents people from using the framework in a "wrong" way, reducing frustration. 🚀